cmd/thanos/sidecar.go

```diff
@@ -45,7 +45,9 @@ import (
 func registerSidecar(app *extkingpin.App) {
 	cmd := app.Command(component.Sidecar.String(), "Sidecar for Prometheus server")
 	conf := &sidecarConfig{}
+	reqLogDecision := cmd.Flag("log.request.decision", "Request Logging for logging the start and end of requests. LogFinishCall is enabled by default. LogFinishCall: Logs the finish call of the requests. LogStartAndFinishCall: Logs the start and finish call of the requests. NoLogCall: Disable request logging.").Default("LogFinishCall").Enum("NoLogCall", "LogFinishCall", "LogStartAndFinishCall")
```

Comment:
`LogErrorFinishCall`

Reply:
log.level=info

LogFinishCall

What if I want to only log gRPC Series method?

Reply:
`log.request.decision`
`log.request.methods`
`log.request.level`

do we want this or be more generic?

Reply:
repeated?

Reply:
We could experiment with! (: 

`log.request` = `.*;LogStartAndFinishCall;info`   or  `grpc`  or  `grpc/Series`

AND we should also look around how nginx or envoy etc other projects are configuring this (:

Reply:
Simple YAML?

Reply:
(some yolo notes from our offline discussion) (:

Reply:
Consensus: We would use a YAML formatting for configuring the options.